src/pages/faretype.tsx

```diff
@@ -1,79 +1,106 @@
+                                    <input
+                                        className="govuk-radios__input"
+                                        id="fareType-period"
+                                        name="fareType"
+                                        type="radio"
+                                        value="period"
+                                    />
+                                    <label className="govuk-label govuk-radios__label" htmlFor="fareType-period">
+                                        Period Tickets
+                                    </label>
+                                </div>
+                                <div className="govuk-radios__item">
+                                    <input
+                                        className="govuk-radios__input"
+                                        id="fareType-return"
+                                        name="fareType"
+                                        type="radio"
+                                        value="return"
+                                        disabled
+                                        aria-disabled="true"
+                                    />
+                                    <label className="govuk-label govuk-radios__label" htmlFor="fareType-return">
+                                        Return
+                                    </label>
+                                </div>
                             </div>
-                        </div>
-                    </fieldset>
-                </div>
-                <input
-                    type="submit"
-                    value="Continue"
-                    id="continue-button"
-                    className="govuk-button govuk-button--start"
-                />
-            </form>
-        </main>
-    </Layout>
-);
+                        </fieldset>
+                    </div>
+                    <input
+                        type="submit"
+                        value="Continue"
+                        id="continue-button"
+                        className="govuk-button govuk-button--start"
+                    />
+                </form>
+            </main>
+        </Layout>
+    );
+};
 
-// eslint-disable-next-line @typescript-eslint/require-await
 export const getServerSideProps = (ctx: NextPageContext): {} => {
-    deleteCookieOnServerSide(ctx, FARETYPE_COOKIE);
+    const cookies = parseCookies(ctx);
+
+    if (cookies[FARETYPE_COOKIE]) {
+        const fareTypeCookie = unescape(decodeURI(cookies[FARETYPE_COOKIE]));
+        const parsedFareTypeCookie = JSON.parse(fareTypeCookie);
+
+        if (parsedFareTypeCookie.errorMessage) {
+            const { errorMessage } = parsedFareTypeCookie;
+            deleteCookieOnServerSide(ctx, FARETYPE_COOKIE);
+            return { props: { errors: [{ errorMessage }] } };
+        }
+    }
 
     return { props: {} };
 };
```

Comment:
Should we change the name of the page and API to fareType?

Reply:
I did but it kept rejecting it... weird. I'll try again now.

Reply:
git config core.ignorecase false

This seems to have solved it.

Reply:
Lovely stuff! Has the API been updated and pushed up as well? Can't seem to see it changed from `faretype.ts` to `fareType.ts`

Reply:
Done also.